src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1213,6 +1215,57 @@ public void readonly() {
     sendCommand(READONLY);
   }
 
+  public void geoadd(byte[] key, double longitude, double latitude, byte[] member) {
+    sendCommand(GEOADD, key, toByteArray(longitude), toByteArray(latitude), member);
+  }
+
+  public void geoadd(byte[] key, Map<byte[], GeoCoordinate> memberCoordinateMap) {
```

Comment:
@HeartSaVioR this is depending on how we implement the `hashcode` and `equals` on the GeoCoordinate bean. If we add the `member` attribute to that class and we modify the mentioned methods accordingly, we could use Set<GeoCoordinate> as an input to the `geoadd` command. 

I'm just thinking about which would be the simplest API / code. If either using Map or Set.